### ISSUE TITLE
Code request

### ISSUE THREAD

**Issue by sebatinoco** [borisbanushev/stockpredictionai#88]
Hi! I loved the notebook that you made. I'm very interested on predicting values using LSTM neural network, would it be possible for me to get the complete code that you used? Thanks in advance!

**Comment by johnwiese** [comment]
> Hi! I loved the notebook that you made. I'm very interested on predicting values using LSTM neural network, would it be possible for me to get the complete code that you used? Thanks in advance!

Yes, me too. Also, do you have the wavelet part that was compared to the FFT.
Amazing work!

**Comment by IISuperluminaLII** [comment]
he says the wavelet part performs similar to FFT, I think you should be alright just with FFT. If you really want to use wavelet, a barebones implementation should work just fine here. Numpy should have a wavelet transformation function I believe
